pkg/operation/botanist/addons/nginxingress/nginxingress.go

```diff
@@ -32,7 +33,18 @@ const (
 )
 
 // Values is a set of configuration values for the nginx-ingress component.
-type Values struct{}
+type Values struct {
+	// ImageController is the container image used for nginx-ingress controller.
+	ImageController string
+	// ImageDefaultBackend is the container image used for default ingress backend.
+	ImageDefaultBackend string
+	// KubernetesVersion is the version of kubernetes for the shoot cluster.
```

Comment:
Nit: Inline with other components
```suggestion
	// KubernetesVersion is the Kubernetes version of the Shoot.
```